llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -1650,7 +1642,6 @@ void VPlanTransforms::truncateToMinimalBitwidths(
             // Add an entry to ProcessedTruncs to avoid counting the same
             // operand multiple times.
             ProcessedTruncs[Op] = nullptr;
-            NumProcessedRecipes += 1;
```

Comment:
Can all the code in the `ifndef` be removed now? Remaining code for `ProcessedTruncs` is only needed to maintain consistent counts, which isn't needed any longer?